restart_process/README.md

```diff
@@ -73,6 +72,14 @@ custom_build_with_restart(
 )
 ```
 
+### Troubleshooting
+#### `failed running [touch /tmp/.restart-proc']`
+If you see an error of the form:
+```
+ERROR: Build Failed: ImageBuild: executor failed running [touch /tmp/.restart-proc']: exit code: 1
+```
+this often means that your Dockerfile user doesn't have permission to write to the file we use to signal a process restart. Use the `restart_file` parameter to specify a file that your Dockerfile user definitely has write access to.
```

Comment:
I worry a bit that "Dockerfile user" isn't really a googleable term. Maybe something like this?

```suggestion
this often means that your Dockerfile user doesn't have permission to write to the file we use to signal a process restart. Use the `restart_file` parameter to specify a file that your Dockerfile user definitely has write access to. You can run `docker run --entrypoint whoami $DOCKER_IMAGE` for your `$DOCKER_IMAGE` to see which user this is, or, e.g., `docker run --entrypoint touch $DOCKER_IMAGE /tmp/.restart-proc` to test permissions.
```

(maybe the last bit is a bit too handholdy?)

Reply:
ehhh kinda handholdy. I just linked to the Dockerfile reference doc for `USER` and if ppl still have questions they can come to us